Add Video Calling/iOSVideo/ContentView.swift

```diff
@@ -0,0 +1,367 @@
+//
+//  ContentView.swift
+//  iOSVideo
+//
+//  Created by Xu Mo Microsoft on 2/23/21.
```

Comment:
Should the files contain the name of the person who created them? Maybe something more generic like Created by Microsoft or Azure Communication Services?

The communication services ios hero sample has the following headers:

//
//  Copyright (c) Microsoft Corporation. All rights reserved.
//  Licensed under the MIT License.
//

Reply:
yes we should remove individual names